recipe/meta.yaml

```diff
@@ -51,6 +51,7 @@ requirements:
   host:
     - libcxx {{ cxx_compiler_version }}  # [osx]
     - llvmdev =={{ version }}
+    - zlib                               # [linux]
```

Comment:
@isuruf, this is needed by `build.sh` before any of the outputs.

My question is: do we _also_ need to add zlib to all of the outputs individually?

Reply:
Currently following a minimal approach to add this only where needed.